deepchem/data/data_loader.py

```diff
@@ -1050,6 +1050,73 @@ def _add_sequence(sequences: np.ndarray,
 
     return DiskDataset.create_dataset(shard_generator(), data_dir)
 
+class FASTQLoader(DataLoader):
+
```

Comment:
We need unit tests for this loader

Reply:
https://github.com/deepchem/deepchem/blob/master/deepchem/data/tests/test_fasta_loader.py